mtags/src/main/scala-2/scala/meta/internal/pc/CompletionProvider.scala

```diff
@@ -344,7 +344,7 @@ class CompletionProvider(
       item
     }
 
-    val result = new CompletionList(items.toSeq.asJava)
```

Comment:
With toSeq and iterator, the list will actually be evaluated only after being passed all the way to JsonRPC (for Scala 2.12)